Closes # 48 Update NEWS.md ahead of first release (#55)
pharmaverse/admiralmetabolic: NEWS.md

# admiralmetabolic 0.1.0

> - Initial package release mainly focused on obesity therapeutic area

Reviewer: ```suggestion
- Initial package release mainly focused on obesity therapeutic area.
```

# admiralmetabolic 0.1.0

- Initial package release mainly focused on obesity therapeutic area

> (e.g. functions for Waist to Hip Ratio, Waist to Height Ratio).

Author: Deleted example.

## Functions

> - The new function "'derive_param_waisthip'" is provided to derive parameters for Waist to Hip Ratio. (#33 &  #52)
> - The new function "'derive_param_waisthgt'" is provided to derive parameters for Waist to Height Ratio. (#33 & #52)

Reviewer: ```suggestion
- The function `derive_param_waisthip()` for deriving Waist to Hip Ratio. (#33 &  #52)
- The function `derive_param_waisthgt()` for deriving Waist to Height Ratio. (#33 & #52)
```

## Documentation

> - New vignette "Creating a Metabolic ADVS ADaM" has been created. (#28) 
> - New vignette "Creating a Control of Eating Questionnaire ADaM" has been created. (#53)
> - New template scripts for ADVS "'ad_advs.R'" has been created. (#35)

Reviewer: ```suggestion
- Vignette "Creating a Metabolic ADVS ADaM". (#28) 
- Vignette "Creating a Control of Eating Questionnaire ADaM". (#53)
- ADVS template scripts `ad_advs.R` which creates a Vital Sign Analysis Dataset with metabolic specific derivations. (#35)
```

Author: Hi @AndersAskeland , I updated quotations and wordings.
Could you take a look again?

# admiralmetabolic 0.1.0

> - Initial package release mainly focused on obesity indicators

Reviewer 2: ```suggestion
- Initial package release mainly focused on obesity therapeutic area
```

Reviewer: This appears resolved :)

> (e.g. Waist to Hip Ratio, Waist to Height Ratio).

Reviewer 2: ```suggestion
(e.g. functions for Waist to Hip Ratio, Waist to Height Ratio).
```

Reviewer 2: I think if we are going to mention the functions then we should mention our templates and vignettes too. Alternatively as this is our first release we could omit mentioning anything like that in the changelog at all. @AndersAskeland what do you think?

Reviewer 2: ok, happy to go with this

Reviewer: @kiwi5104679 What are your thoughts on this?

Author: @AndersAskeland , @manciniedoardo , thank you.
I think it's better to have subsections.
I'll add "Functions" and "Documentation".

Author: Hi @AndersAskeland , @manciniedoardo.
I've updated NEWS.md.
I noticed that {admiral} and other related packages use "New Features" subsection to describe new functions instead of "Functions", so I added "New Features" subsection in {admiralmetabolic}.
Please let me know if it's better to use "Functions".
![image](https://github.com/user-attachments/assets/42182f79-bfac-4b3e-bac9-f08c7fd5b8aa)
![image](https://github.com/user-attachments/assets/8b71aa47-5473-4344-b172-0c768a4a808c)

Author: In addition, should we have hyperlinks in changelog?
![image](https://github.com/user-attachments/assets/110536b5-85db-42bc-914d-38979224952e)

It's user friendly to have hyperlinks, but I'm concerned that maintenance may be difficult.

Author: Exactly.
Updated subsection to "Functions".
